### DISCUSSION TITLE
Preset Zoom Settings?

### DISCUSSION THREAD

**Discussion by IkeEisenhauer** [xournalpp/xournalpp#4639] [Q&A]
Is it possible to have preset Zoom settings?

Use case:

Depending if I am showing a landscape presentation PDF, or notepaper, I need to adjust the zoom with different document types to match a sharing window in OBS. Normal size and Best Fit do not quite cut it.

Right now I have to manual adjust the zoom each time to a known level using a slider. If there is a method to store preset zoom settings or manual enter it directly without the slider, I am overlooking it. 

Does that feature exist and just am too new?

**Comment by CarlNor** [comment]
I'm having similar problems. Or rather I think presentation mode works pretty well for recording with obs. But when stepping back in the presentation the position of the page on the screen shifts. Which means it is missaligned with the obs capture window.

**Comment by rolandlo** [comment]
@IkeEisenhauer If you close a .xopp-file or .pdf file the zoom level should get stored in a metadata file (inside the config folder, e.g. `~/.config/xournalpp/metadata` in case of Linux). When you open the same file again next time, the zoom level should get restored. I hope that helps.

@CarNor That's annoying, but it will get fixed in #4434 eventually.

**Comment by IkeEisenhauer** [comment]
Yes I noticed that as well, a kludgy work around is to add a copy of the
first slide, and then advance down, set the capture window based on that
one, then every time you go "back in presentation" just go one slide too
far and then page down.

Annoying but it does work

**Comment by IkeEisenhauer** [comment]
Problem is I tend to open a new PDF each lecture...so this really doesn't
work as a long term solution.
